content/lua/concepts/mathematical-library/terms/modf/modf.md

```diff
@@ -0,0 +1,81 @@
+---
+Title: 'modf()'
+Description: 'Used to extract the integral and fractional parts of a given number.'
+Subjects:
+  - 'Computer Science'
+  - 'Game Development'
+Tags:
+  - 'Arithmetic'
+  - 'Functions'
+  - 'Math'
+  - 'Numbers'
+CatalogContent:
+  - 'learn-lua'
+  - 'paths/computer-science'
+---
+
+The **`math.modf()`** function is a part of the Lua `math` library and is used to split a given number into _its integral and fractional parts_. It rounds the number down to the nearest integer and returns both the integer part and the fractional part as separate values.
+
+> **Note:** `math.modf()` rounds the number down and always returns a value equal to or less than the given number.
+
+## Syntax
+
+As `modf()` is a method that is a part of the standard Lua `math` library, it must be called as `math.modf()`.
+
+```pseudo
+integerPart, fractionalPart = math.modf(number) -- Returns the intergal and fractional part
+```
+
```

Comment:
````suggestion
integerPart, fractionalPart = math.modf(number)
```

`math.modf()` returns the integral part of `number` in `integerPart` and the fractional part in `fractionalPart`.

````

Prefer the explanatory text outside the code block.